### DISCUSSION TITLE
DA in observation space

### DISCUSSION THREAD

**Discussion by cbblanke** [NASA-LIS/LISF#894] [Q&A]
Can someone inform me on the status of doing data assimilation in observation space?  For example, if you have 3-km grid cells and 30-km observations, you should be able to adjust all the model cells in the observation up or down (by the same increment, assuming background errors are the same) to satisfy the observation.  If this has been implemented, where is it described?  Is there a testcase?

**Comment by jvgeiger** [comment]
For others' reference see Discussion #1157.
